pkg/clustermanager/cluster_manager.go

```diff
@@ -999,6 +1004,17 @@ func (c *ClusterManager) applyResource(ctx context.Context, cluster *types.Clust
 	return nil
 }
 
+func (c *ClusterManager) applyEksdManifest(ctx context.Context, cluster *types.Cluster, eksdComponents *cluster.EksdManifests) error {
+	if err := c.Retrier.Retry(
+		func() error {
+			return c.clusterClient.ApplyKubeSpecFromBytesWithNamespace(ctx, cluster, eksdComponents.ReleaseManifestContent, constants.EksaSystemNamespace)
+		},
+	); err != nil {
+		return fmt.Errorf("error applying eksd release manifest: %v", err)
```

Comment:
adding the keyword `error` adds a new redundant keyword and can make an error message look confusing. can we remove the `error` word?